x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/EsqlTestUtils.java

```diff
@@ -119,10 +118,53 @@ public boolean isIndexed(String field) {
 
     public static final TestSearchStats TEST_SEARCH_STATS = new TestSearchStats();
 
-    public static final EsqlConfiguration TEST_CFG = configuration(new QueryPragmas(Settings.EMPTY));
-
     public static final Verifier TEST_VERIFIER = new Verifier(new Metrics());
 
+    private static final Map<String, Map<String, Column>> TABLES;
```

Comment:
Moved this up since the statics are initialized in declaration order - without this change the TEST_CONFIG always receives a NULL table since the static block isn't yet ran.